K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -73,143 +73,150 @@ public class AggregationConverterTests
 
         private const string NoAggAggregation = @"
-        {""aggs"": { 
-            ""2"": {
-                ""noagg"" : { 
-                    ""field"" : ""metric"" 
-                    } 
-            }
-        }}";
+            {""aggs"": { 
+                ""2"": {
+                    ""noagg"" : { 
+                        ""field"" : ""metric"" 
+                        } 
+                }
+            }}";
 
-        private static readonly Aggregation ExpectedValidDateHistogramAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidDateHistogramAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new DateHistogramAggregation {
-                            FieldName = "timestamp",
-                            FieldAlias = "_2",
-                            Interval = "1m",
-                            Metric = "count()",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new DateHistogramAggregation
+                    {
+                        Field = "timestamp",
+                        Key = "2",
+                        FixedInterval = "1m",
+                        TimeZone = "Asia/Jerusalem",
+                        MinimumDocumentCount = 1,
                     },
+                    SubAggregations = new AggregationDictionary(),
                 },
+            },
         };
 
-        private static readonly Aggregation ExpectedValidTermsAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidTermsAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
             {
+                ["2"] = new AggregationContainer
                 {
-                    "2", new Aggregation() {
-                        PrimaryAggregation = new TermsAggregation {
-                            FieldName = "DestCountry",
-                            FieldAlias = "_2",
-                            Metric = "count()",
-                            SortFieldName = "_count",
-                            SortOrder = "desc",
+                    PrimaryAggregation = new TermsAggregation
+                    {
+                        Field = "DestCountry",
+                        Key = "2",
+                        Order = new TermsOrder {
+                            Field = "_count",
+                            Sort = "desc",
```

Comment:
IMO, sortOrder was more self explanatory

Reply:
I agree. Fixed the code to use SortField and SortOrder.